Labs/Alyssa_Giedd_Lab4.ipynb

```diff
@@ -277,26 +277,289 @@
DPMwNRZFm311AZw/K52OTzMYaABZhLlQrX5kAXtyvdELX4cmr69dJm3bCn7u0M+9yIj3OOUR9rKSal9v+HsGJk9YyLmVS1S8BhlPSprlWSzzS1tjUr+IV5r4xty5+qfBEB2TworvYTXWtaEi0AYUnJ6ltDOYEYMMibYuP6oBGqhq3C8fhaCL6h2F/QVPLiuRw2Ki8TOK0el7GpVHFSJ8jkgTIRdn4Bs45pJyMkifhS7Ud87HF31RTTTXVVFNN9fhycOdskBETos/9KGtlhX/qZ/4tswwuyDjmgw0FTINnz0HtK/sPIhAU6otediE/B2fZIKd8MrE+3a88zo5g7SBaD9LTGQgZXcAhy6B10LDx4IItYb2iBexhRHcjFDYGXjKXzmQAcIbyNKHReIre3G3Jbmk0C08GgpmZSXGxy9DMystw0IQGpWRj4iy0nz33hgm7tLuwMW6HFptqY9rkozp+BRfQle6seS69mQjQGprIS1SkzClLiAv2Dy6YJYBTc1xUjzRe6r8Kiv4bPbQ4bdCZDeKdx9d907eAyaveORyGA77/677LJuNt2lpa54vu1iZCADY5k8I1HkKCCyLeV34juYIeZ/hrERa4inoxufPFJD7Z+WIAyigXzl6kP9Gy+JRPGoAptsoUrxNtuK5WWIQ52oGWdgr96UVRFGFZx7W5pBGhIVVrFa8sF+Zh2Bst7JyLMrMTkHzGqyj0u6uwRCkFx3zAXX9norNlWBqCxcGMQwi/tAgdCh/ySp0/JJDzYdirFfIRL7pbOOfMkpoIGSC0Pa/OJTO1OybUCuAibZdI1VRTTTXVVFNN9arqvEBchDkcPH7HH/j9oqdVnUiXT/hzX/OvYa79D5fJV2Fpgw71Jsz5OAwHa3TH1HVmotAm2RatvrI+ahxvMfM1Pnt6z3qyw3DAIiwscJoxDQBM//K0eqrN/4MiLM4yZoiisL9YhiXmfmEJ8pJUL6qcbb81FIbOXwCM/tYOrS1jvXPi/FXyBfuHvRLtlHdpp3ET8vwzdU8d2xWv4koc3ZQ2FlzA8+65oWKlFFyrkROPIwcVyf3zWIWV0c04UErP5e2Ycfg7qWaHi3S6xQ4lofIV9mlvZlGvwgzpjR5asorA2+Fk0+ovfvInTFR+GI4opWjCesLcgnCEJvSsfnLmS2qey0nhx8oLt5KPRfRgHFLIn3WarnpSiz2xC34wtwgOUeK8kE0bQhqac04cv9T7OqmDhtcTjAnxFHnxokxlwHW1Vi3N3qbtU+4U3YiW3wIIte0wHO098XFkYl9gFa/sYiAyQX7mSQczQY9mZjpA+hsF+eNEV24dJPH2bBNI3iYDoJj2Gn28sDa+rlYjo4XZ3+UaltXGuR1OyMg2SFLDQgSKF/BUU0011VRTTfXqKukC8pQ7FGT83F/7H02QnYv8O72u1ra0bXyDF92tLW0pJCeFaOZnWIQFFmGuTBFnTTHpW5WvlFkRVRif0YTGsvkqH80OmZEJjW/OzlyKqnjnzJaXr6VTehVZPOyTdmln4n6yc6ILZpTErBX2NPMwR19E10EHtV7z5c5mBbIo5wKex4zRGyfVAxFxaXxj7mpjVInOZbWrsVWqf+Uj3uveNwcwDoi1Gh6Jc21jA1VRe+hcMu7SnX2+1Mcc1RiByA6Aizw/LsDZj4neWBCaAYMu4R+/PH6jhxa6ePHDf949B4ALEb3lngCW3XEd1yaGZ8ZLN+JHcroFcM53GelP+pzUx9tZQ8ym3StFq/Y1roKERfIEJfz5d9rPGmrgnTd0Z6auYgzs8c4pUiEuGzKQnNAOJ5vqGeqzqa5xHI7Y6nt0mppKj3LmnzDMyCvEyuR4CuLJWxxnmtB4oC89FmGBfdrbBWtfVjo4SUZLY17ptHBexSsswxIP6QDnnLiQqZj/lDvs0gNSTripro37SUMEDjlGD0M2yhgpdUJ/SyaGW4SFfSaEUPklOdVUU0011VRTPa4KxExolx5sSUm7XwC2tD0MB9z198aK4UKy8tIfMGSaTIo+izspqfMMrfSKFkhKfI9KnVnHAYn3aSvB3krBMidXaH5eFiRHAqln6taqrBelOHlIj9KVTkO+xYCJ0gP2kRlFjX86dQtLdhwAQW8OwwFtbi2OwTuPudLCxGgg2sBDytcqXhlzJJVBqPTap3K4YZFe1+UO27S14PHgIjbVRoYHH+14kGIng5T0fd456xW5HGbvBpxD2zmgMZJDrJRnFyZNzMNh3yZh7EJZGx+bj1ofmtPinHsLwB8H8BXj+5dS/qVHP/srKArnnTb4la+QczaYsEKFHj1QYJt+bvOJIND6jY5ZTWzQKcJAGzg24K0KxwCYJa+Hw8l1YgKgTlmLsDAbOp5sgAjL3pm9JSFDfm4isb702IRr7NUmmJP/fthjFVdmvXfKnbmBDbp9uApL08/M3Vy5nh53/Z1BfkL1EvcOojf9IINJKQXRCzokwY0ZAUGHh4wBwleUYybWxkVFXkRg+tITKcYpnyz4kQhYm0940d1iU13rcdjhulqbBzo3IAyr5HsFoNqW3uBegUuF6sYMnVZtCBdB3j+yIEsSMHrOyplqqqmmmmqqN6Ve5x7M4Rww7YtHX5ItXhnwyO19cUIXCyGazS8g1DKJpxB6+qCWw+KSJY30fdrirXqJUERXwp7nRX8L5riw31iGJbwT+lUVKtz293hS31hAIx1gAaAohS2X3iIajsMRN9UNbvtbrKs1Tvlk9+eiWHQn5+w/4CxAp1Zm5s9UKPZklC0MgDFy2CPNfSMDVTkY+hMQ7OepvWFPS82NLKQPWMUV2qE1OYB3Hn3usYpXuApLvOhv4by3wYk21aSJVUFE+945zPzcaHMO58BMGhH0ucfz/vmFyyufn0ON02DOZVyafonH8TH1MkjLjwC4BvC/APjvR79+w4vIh4RJig1vn+XkqPw5rVOs84R3+JAEUmM6PC+cUkRj8o1/6bt1Ej+CuSxMCW3CmS7W6RYguIBjbkfOXtGae8JqKSdrtMd
```

Comment:
Here you can adjust the sublots position for an easier read, this is how I did mine: plt.subplots_adjust(bottom=0.1, right=0.8, top=0.9, hspace=.3)